RecoTracker/LSTCore/interface/LST.h

```diff
@@ -16,14 +16,15 @@ namespace SDL {
   template <typename>
   class LST;
 
-  template <>
-  class LST<SDL::Acc> {
+  template <typename TAcc>
+  class LST {
```

Comment:
is it possible that there is a mixup in symbols somewhere?
The previous build explicitly instantiated `LST<Acc>`  and implemented that. Is it possible that some code path now makes another `LST<LooksLikeAcc>` that's not already the same as `SDL::LST<Acc3D>`

 `template <typename> class LST;` above should be removed. (I didn't check other

Reply:
Hmm I'm not sure how that would happen, but I'll take a look. I just left a comment with another hypothesis.

Reply:
I don't think this could happen since the implementations in `LST.dev.cc` are explicitly for `SDL::LST<Acc3D>`